code/datums/diseases/_disease.dm

```diff
@@ -116,8 +135,19 @@
 
 
 /datum/disease/proc/Copy()
+	//note that stage is not copied over - the copy starts over at stage 1
+	var/static/list/copy_vars = list("name", "visibility_flags", "disease_flags", "spread_flags", "form", "desc", "agent", "spread_text",
```

Comment:
VV lock this just in case.

Reply:
You cannot access static proc vars from VV.

Reply:
Oh right, its local, carry on.